scripts/install-memkind.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
I get an error when attempting to run this script on both Ubuntu and CentOS (even after installing the dependencies mentioned [here](https://docs.pmem.io/persistent-memory/getting-started-guide/installing-ndctl#1-install-prerequisites)). Are there other steps that need to be taken before running the script?

Reply:
you need to install Memkind [dependencies](https://github.com/memkind/memkind#dependencies).
Also please install `libkmod-dev`